pkgpanda/__init__.py

```diff
@@ -754,6 +801,19 @@ def symlink_all(src, dest):
                         uid = sysusers.get_uid(package.username)
                         check_call(['chown', '-R', str(uid), state_dir_path])
 
+            if package.sysctl:
+                service_names = _get_service_names(package.path)
+
+                if service_names is None:
+                    raise ValueError("service name required for sysctl could not be determined for {package}".format(
+                            package=package.id))
+
+                for service in service_names:
+                    if service in package.sysctl:
+                        dcos_service_configuration["sysctl"][service] = package.sysctl[service]
+
+        write_json(self.__dcos_service_configuration_json, dcos_service_configuration)
```

Comment:
this should either be written inside of `/etc.new` (With a check that no such file already exists), or should be written with as a `.new` and swapped like we do environment.new -> environment.